lib/ui/screens/types/typeCard.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex/configs/types.dart';
+import 'package:pokedex/ui/screens/types/typeSheet.dart';
```

Comment:
The `typeCard.dart` and `typeSheet.dart`, etc is not a Dart suggested name format. Please change it to `type_card.dart` and `type_sheet.dart`